src/test/java/ch/wisv/areafiftylan/TicketRestIntegrationTest.java

```diff
@@ -98,32 +98,41 @@ public void testAddTransfer_Owner(){
     }
 
     @Test
-    public void testAddTransfer_Receiver(){
-        addTicketTransfer(ticketReceiver.getUsername(), ticketReceiverCleartextPassword)
-                .then()
-                    .statusCode(HttpStatus.SC_FORBIDDEN);
+    public void testAddTransfer_OwnerDifferentCase() {
+        String token = addTicketTransfer(user.getUsername().toUpperCase(), userCleartextPassword).then()
+                .statusCode(HttpStatus.SC_OK).extract().path("object");
 
-        Assert.assertTrue(tttRepository.count() == 0);
+        TicketTransferToken ttt = tttRepository.findByToken(token).orElseThrow(() -> new TokenNotFoundException(token));
+
+        Assert.assertTrue(ttt.isValid());
+        Assert.assertTrue(ttt.getTicket().getOwner().equals(user));
+        Assert.assertTrue(ttt.getUser().equals(ticketReceiver));
     }
 
     @Test
-    public void testAddTransfer_Outsider(){
-        addTicketTransfer(outsider.getUsername(), outsiderCleartextPassword)
-                .then()
+    public void testAddTransfer_Receiver() {
+        addTicketTransfer(ticketReceiver.getUsername(), ticketReceiverCleartextPassword).then()
                 .statusCode(HttpStatus.SC_FORBIDDEN);
 
         Assert.assertTrue(tttRepository.count() == 0);
     }
 
     @Test
-    public void testDoTransfer_Anon(){
+    public void testAddTransfer_Outsider() {
+        addTicketTransfer(outsider.getUsername(), outsiderCleartextPassword).then().statusCode(HttpStatus.SC_FORBIDDEN);
+
+        Assert.assertTrue(tttRepository.count() == 0);
+    }
+
+    @Test
+    public void testDoTransfer_Anon() {
         TicketTransferToken ttt = addTicketTransferGetToken();
 
         given().
-        when().
+                when().
```

Comment:
https://github.com/AreaFiftyLAN/lancie-api/pull/214#issuecomment-217851962
I think it is because the auto formatter is off.